src/task_queue/nudges/user_event_nudge.py

```diff
@@ -107,37 +107,41 @@ def update_last_notification_dates(email):
 
 
 def is_event_eligible(event, community_id, task=None):
-    now = timezone.now()
-    settings = event.nudge_settings.filter(communities__id=community_id).first()
-
-    # it it doesn't exist, then create, don't save an instance of NudgeSettings
-    if not settings:
-        settings = EventNudgeSetting.objects.create(event=event, **DEFAULT_EVENT_SETTINGS)
-        settings.communities.add(community_id)
-
-    if settings.never:
+    try:
+        now = timezone.now().date()
+        settings = event.nudge_settings.filter(communities__id=community_id).first()
+        
+        # it it doesn't exist, then create, don't save an instance of NudgeSettings
+        if not settings:
+            settings = EventNudgeSetting.objects.create(event=event, **DEFAULT_EVENT_SETTINGS)
```

Comment:
Suggest change this to settings = EventNudgeSetting(event=event, **DEFAULT_EVENT_SETTINGS)
so as not to save unnecessary default settings to DB

Reply:
Done. Thanks